src/corecel/random/params/XorwowRngParams.cc

```diff
@@ -43,39 +43,40 @@ XorwowRngParams::XorwowRngParams(unsigned int seed)
  */
 auto XorwowRngParams::get_jump_poly() -> ArrayJumpPoly const&
 {
-    static ArrayJumpPoly const jump = {
-        {{0x00000002u, 0x00000000u, 0x00000000u, 0x00000000u, 0x00000000u},
-         {0x00000010u, 0x00000000u, 0x00000000u, 0x00000000u, 0x00000000u},
-         {0x00010000u, 0x00000000u, 0x00000000u, 0x00000000u, 0x00000000u},
-         {0x00000000u, 0x00000000u, 0x00000001u, 0x00000000u, 0x00000000u},
-         {0xbebd3534u, 0x7064f5bcu, 0x20be29ebu, 0x536d5b32u, 0x063a0069u},
-         {0xed64ec08u, 0xafc48684u, 0xd81c59eeu, 0x1640314fu, 0x2bf0ccefu},
-         {0x9d10e028u, 0xee56d79cu, 0xfb1b3286u, 0x0f747418u, 0x26f9476du},
-         {0x3f490634u, 0x9ae593fcu, 0x1a95bb6bu, 0xda10a3fcu, 0xa3abaf54u},
-         {0xfb9680e9u, 0xbdaba0b2u, 0x3986540fu, 0x23fe6cccu, 0x0994e82fu},
-         {0x32da6db4u, 0x80135829u, 0x3abd4734u, 0x2060c3f9u, 0x38b2dd97u},
-         {0x0c5b636fu, 0x4407a814u, 0x60204515u, 0x5bd4dbd2u, 0x2509eeb5u},
-         {0x21e9179du, 0x2b57aa94u, 0x5f06e1fcu, 0x6735dc98u, 0x9aa0181fu},
-         {0x793adf2bu, 0x3d3e75c8u, 0x0b091758u, 0x9deb3f50u, 0xbcd116ecu},
-         {0xfa8fc346u, 0x694921b1u, 0xf2bd4c48u, 0x8b05ae1bu, 0x2de7aee2u},
-         {0xf144e1f3u, 0xb9b4b3c4u, 0x8222f622u, 0x9072105au, 0x66083550u},
-         {0xbd734cf1u, 0x9254a905u, 0xfb38236du, 0x11e62fd3u, 0xf9e7d21eu},
-         {0x943999e4u, 0xc05db913u, 0x4e4010f3u, 0x9b865d3du, 0x0fd64174u},
-         {0x19eb1bbbu, 0xbaea3750u, 0x0fa8f206u, 0xd49dd019u, 0x9b3bccafu},
-         {0xc97b5642u, 0x2ebac13du, 0xbfe04058u, 0x2c6a7132u, 0x576780a5u},
-         {0x0ac5eea9u, 0xa37bfcd3u, 0x790ec91du, 0xba339dbcu, 0xc83db5cdu},
-         {0xa33b53ffu, 0x1ce9360du, 0x4727f89bu, 0x05eacbcdu, 0x01632278u},
-         {0x22b4f98cu, 0xd23a7f5au, 0x8d420eddu, 0xeadda806u, 0xcfd2a002u},
-         {0xf66ad52bu, 0x3ab8e3d7u, 0x1e8352a4u, 0xe44a8605u, 0x6c106869u},
-         {0x79a31c08u, 0xc28d5d18u, 0x91649708u, 0x8b1bb8f8u, 0xf9158a86u},
-         {0x670f870eu, 0xa7bb9766u, 0xef013c78u, 0xeb4a1373u, 0x256f3323u},
-         {0xf333af18u, 0x3b4e266bu, 0xa872663eu, 0x3888cd82u, 0x4daf13ecu},
-         {0x75689dc9u, 0x036bf3a9u, 0x64ce979cu, 0x3bdff14fu, 0x51e43048u},
-         {0xef06fe75u, 0xefd6d1d8u, 0x09319075u, 0x69d568f5u, 0x5b2bd898u},
-         {0xf05ae255u, 0xc4df4ca0u, 0xf032420cu, 0xf44ae9f0u, 0xe0298de2u},
-         {0x02308bc9u, 0xdbe74deeu, 0xf4c5fe7du, 0xaac571a1u, 0xaa1f5f8bu},
-         {0x6e8043cau, 0x0ed24ff6u, 0x1e668b6au, 0x538fc45fu, 0x4bfb509du},
-         {0x9f564f7bu, 0x543973e4u, 0x9b33ee2cu, 0x149df73au, 0x58f31585u}}};
+    static ArrayJumpPoly const jump = {{
```

Comment:
Is `'{{'` still required instead of `'{'`?

Reply:
Yes unfortunately; it's a consequence of using an Array inside an Array; they both have variable-argument templates, so the compiler can't deduce it. Using the extra braces initializes using the `T const (&)[N]` constructor rather than the varargs constructor.

Alternatively we could write:
```c++
    using JP = JumpPoly;
    static ArrayJumpPoly const jump = {
        JP{0x00000002u, 0x00000000u, 0x00000000u, 0x00000000u, 0x00000000u},
        JP{0x00000010u, 0x00000000u, 0x00000000u, 0x00000000u, 0x00000000u},
....
```
but I think just including the extra braces is easier.